tss-esapi/src/context/tpm_commands/asymmetric_primitives.rs

```diff
@@ -75,8 +75,63 @@ impl Context {
         }
     }
 
-    // Missing function: ECDH_KeyGen
-    // Missing function: ECDH_ZGen
+    /// Generate an ephemeral key pair.
```

Comment:
It would be nice if you could provide an example in the documentation of the added context method.

Reply:
Yes, definitely! I've added more comprehensive doc tests for key gen and z-gen.